esp-hal-embassy/src/executor/thread.rs

```diff
@@ -72,9 +72,7 @@ impl Executor {
         multi_core,
         doc = r#"
 
-    This will use software-interrupt 3 which isn't
-    available for anything else to wake the other core(s).
-    "#
+This will use software-interrupt 3 which isn't available for anything else to wake the other core(s)."#
```

Comment:
```
warning: could not parse code block as Rust code
  --> src\executor\thread.rs:70:5
   |
70 | /     /// Create a new Executor.
71 | |     #[cfg_attr(
72 | |         multi_core,
73 | |         doc = r#"
...  |
76 | |     available for anything else to wake the other core(s).
77 | |     "#
   | |______^
   |
   = note: error from rustc: prefix `isn` is unknown
   = note: `#[warn(rustdoc::invalid_rust_codeblocks)]` on by default
```

Reply:
where did that show up for you? Don't think I've seen this before

Reply:
I built the documentation for esp-hal-embassy, which we don't usually do.